### PR TITLE
Added functionality to move a problem from one contest to another

### PR DESCRIPTION
**Pull request by RoYaLBG**

### CODE REVIEW

**Comment by NikolayIT** [comment]
Could you please pull request the same change in https://github.com/NikolayIT/OpenJudgeSystem :-)

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Edit.cshtml [hunk 1]
```diff
@@ -118,6 +116,40 @@
...
             </div>
+            <br/>
+            <div class="row">
+                <div class="editor-label col-xs-3">
+                    <div class="pull-right">
+                        @Html.LabelFor(m => m.ContestId)
+                    </div>
+                </div>
+                <div class="col-xs-5">
+                    @(@Html.Kendo().ComboBox()
```
Comment (vladislav-karamfilov) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Edit.cshtml]: You don't need the second `@`.

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Edit.cshtml [hunk 2]
```diff
@@ -118,6 +116,40 @@
...
+                    .Name("ContestId")
```
Comment (vladislav-karamfilov) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Edit.cshtml]: Indentation.

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Edit.cshtml [hunk 3]
```diff
@@ -246,5 +278,6 @@
 
 @section scripts {
     <script src="~/Scripts/Administration/Problems/problems-edit.js"></script>
+    <script src="~/Scripts/Administration/Problems/problems-index.js"></script>
```
Comment (KristianMariyanov) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Edit.cshtml]: Are we need all this JavaScript in `problems-index.js`?

**Pull request merged**